canvas_modules/common-canvas/package.json

```diff
@@ -36,6 +36,8 @@
     "@codemirror/state": "^6.4.1",
     "@codemirror/view": "^6.28.6",
     "@elyra/pipeline-schemas": "^3.0.96",
+    "@tanstack/react-table": "8.20.6",
```

Comment:
these should be using `^` so consuming applications can get the latest versions.